docs/modules/ROOT/pages/explanations/crossplane_service_broker.adoc

```diff
@@ -327,22 +340,14 @@ Service Plan (Composition)::
 
 Service Instance (XR)::
 
-* tbd
+Currently none.
```

Comment:
The cluster name is stored in a label: `service.syn.tools/cluster`

Reply:
yup, but that's a label and this part here is about annotations :) the cluster name is above mentioned already :)